heartbeat/ibm-cloud-vpc-move-fip.in

```diff
@@ -0,0 +1,248 @@
+#!@PYTHON@ -tt
+# ------------------------------------------------------------------------
+# Description:  Resource Agent to move an IBM Cloud  Floating IP (FIP)
+#		From one Virtual network Interface to another
+#
+# Authors:      Eran Gampel
+#
+# Copyright (c) 2025 International Business Machines, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+import subprocess
+import ipaddress
+import os
+import sys
+import textwrap
+
+OCF_FUNCTIONS_DIR = os.environ.get(
+    "OCF_FUNCTIONS_DIR", "%s/lib/heartbeat" % os.environ.get("OCF_ROOT")
+)
+
+sys.path.append(OCF_FUNCTIONS_DIR)
+
+
+try:
+    import ocf
+except ImportError:
+    sys.stderr.write("ImportError: ocf module import failed.")
+    sys.exit(5)
+
+try:
+    import ibm_cloud_fail_over
+except ImportError:
+    sys.stderr.write("ImportError: import ibm_cloud_fail_over module import failed.")
```

Comment:
You can use `ocf_exit_reason()` or `ocf.logger.error()` here.

Reply:
makes sense , i will send a PR to fix it in the cr-vip IBM cloud plugin